Umbraco-Uno/Creating-Content/How-to-Set-up-a-Blog with-a-Feed/index.md

```diff
@@ -0,0 +1,51 @@
+
+---
+versionFrom: 8.0.0
+---
+
+# The Blog
+In this guide you will learn, to set up a blog with a feed that contains the blog posts. 
```

Comment:
In this guide you will learn how to set up a blog with a feed that contains the blog's posts.